gyp/xcode_emulation.py

```diff
@@ -1405,7 +1405,7 @@ def XcodeVersion():
     if version:
       version = re.search(r'^(\d{1,2}\.\d(\.\d+)?)', version).groups()[0]
     else:
-      raise GypError("No Xcode or CLT version detected!")
+      raise GypError("No Xcode or CLT version detected!  {}".format(version))
```

Comment:
I'll migrate the "new" way to detect
https://github.com/refack/GYP/blob/ac5703c4fa16c75f725b90e83d52a9e0821daf3f/testlib/TestMac.py#L56-L64